### ISSUE TITLE
Narrative tables (md or html) need styling

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#887]
(Moved out of #883 for clarity)

@emmahodcroft:

I currently can't seem to get tables (either Markdown or HTML) working in the auspiceMainDisplayMarkdown part working. This might be nice for formatting lists of links with images, for example. Or lists of previous situation reports (or other languages)

@jameshadfield:

I presume HTML tables don't work because of https://github.com/nextstrain/auspice/blob/master/src/components/narrative/MainDisplayMarkdown.js#L136 (this code was copied from the footer markdown parsing). If markdown tables don't work then there may be an option we can add to https://github.com/nextstrain/auspice/blob/master/cli/server/parseNarrative.js#L13 to improve this.

_UPDATE: Thanks to @CameronDevine (#924) tables now display in `auspiceMainDisplayMarkdown` sections. These things are still needed:_

- [ ] The tables in `auspiceMainDisplayMarkdown` sections need to be styled appropriately
- [x] "Normal" (sidebar) narrative text should be able to display tables (low priority)

**Comment by jameshadfield** [comment]
To reproduce / work on a fix:

* Install auspice from source, as per https://nextstrain.github.io/auspice/introduction/install or similar
* Obtain example datasets + narratives via `npm run get-data && npm run get-narratives`
* Add markdown tables to a narrative which is now present in `./narratives`
* Test via `auspice develop --verbose`

**Comment by CameronDevine** [comment]
I was able to get tables in the `auspiceMainDisplayMarkdown` section working. However, I think some CSS work may also be necessary. As this is not my area of expertise, I will leave this work to someone else.

**Comment by jameshadfield** [comment]
Can't seem to get this working - could you paste an example narrative markdown file (or section of) which shows this working?

**Comment by CameronDevine** [comment]
@jameshadfield Tables in the sidebar also work (This was working before #924). For example:

```
### Further Reading:

| test | table |
| --- | --- |
| 1 | 2 |
| 3 | 4 |
```

![image](https://user-images.githubusercontent.com/10394896/76251331-14686a80-6204-11ea-9f39-9c2c8f9df37e.png)

**Comment by joachimschmidt557** [comment]
I would be happy to work on styling tables.

**Comment by jameshadfield** [comment]
Closed by #966 -- thanks @joachimschmidt557 🎉
